README.md

```diff
@@ -61,6 +61,7 @@ The Monitor Plan API uses a number of environment variables to properly configur
 | currentUser | EASEY_MONITOR_PLAN_API_CURRENT_USER | {} | Configurable |
 | apiHost | EASEY_API_GATEWAY_HOST | api.epa.gov/easey/dev | Configurable |
 | authApi.uri | EASEY_AUTH_API | https://api.epa.gov/easey/dev/auth-mgmt | Configurable |
+apiHost | EASY_CONTENT_API | https://api.epa.gov/easey/dev/content-mgmt | Configurable
```

Comment:
Need to change the typescript variable name and environment variable name
| contentApi.uri | EASEY_CONTENT_API | https://api.epa.gov/easey/dev/content-mgmt | Configurable |